tests/encoders/mongodb.py

```diff
@@ -1,5 +1,7 @@
 """Tests for mongodb encoders"""
 import pytest
+from bson.decimal128 import Decimal128
+from decimal import Decimal
```

Comment:
What for, if not used?

Reply:
Oops, I stashed tests and forgot to unstash. No idea why imports were commited though.. Fixed in 7c1449cf094951c2e4088178ae17bf96dc9f6db8